tests/clients/test_mqtt_client.py

```diff
@@ -103,7 +103,7 @@ def on_message_second(mqttc, obj, msg, properties=None):
                                         callback=on_message_first)
         self.mqttc.message_callback_add(sub=second_topic,
                                         callback=on_message_second)
-        mqtt_broker_url = urlparse(settings.MQTT_BROKER_URL)
+        mqtt_broker_url = urlparse(str(settings.MQTT_BROKER_URL))
```

Comment:
Yep, but the validators are not invoked in `TestSettings`, so the urls here are objectes of pydantic `AnyHttpUrl` or `AnyUrl`. I remove the `urlparse` right now, because it seems not necessary